tests/test_db.py

```diff
@@ -24,20 +29,20 @@ def classify_pauli(arr: npt.NDArray[np.complex128]) -> tuple[int, int]:
         Returns
         -------
             ind : tuple
-                tuple containing (pauli index, sign index)
+                tuple containing (Pauli string, sign)
```

Comment:
`CliffordMeasure` (or `PauliMeasurement`).

Reply:
Fixed in bb6e8489d114667bd5389c1a05c14fd43d6ac1c2 .